### PR TITLE
feat(vote): remove votes of removed controllers PE-7653

### PR DESCRIPTION
**Pull request by arielmelendez**

### CODE REVIEW

File: process/src/main.lua [hunk 1]
```diff
@@ -154,41 +159,93 @@ end, function(msg)
...
-	msg.aoEvent:addField("Proposal-Number", proposal.proposalNumber)
-	msg.aoEvent:addField("Proposal-Name", proposalName)
-	msg.aoEvent:addField("Proposal-Type", proposal.type)
-	msg.aoEvent:addField("Yays-Count", yaysCount)
-	msg.aoEvent:addField("Yays", utils.getTableKeys(proposal.yays))
-	msg.aoEvent:addField("Nays-Count", naysCount)
-	msg.aoEvent:addField("Nays", utils.getTableKeys(proposal.nays))
-	msg.aoEvent:addField("Pass-Threshold", passThreshold)
-	msg.aoEvent:addField("Fail-Threshold", failThreshold)
+	local failThreshold = math.max(controllersCount - passThreshold + 1, 1)
```
Comment (arielmelendez) [inline process/src/main.lua:217]: This was an interesting corner case that I picked up while testing this. All existing tests pass with the change. I need to double check why the existing tests didn't catch the corner case.

Comment (arielmelendez) [reply process/src/main.lua:217]: I think I can actually simplify the logic to remove the math.max now...

Comment (arielmelendez) [reply process/src/main.lua]: 70d958a

**Review by fedellen — APPROVED** [review]

**Review by atticusofsparta — APPROVED** [review]
